### ISSUE TITLE
2021-06

### ISSUE THREAD

**Issue by qiaojy19** [qiaojy19/q-Daily#13]

**Comment by qiaojy19** [comment]
# Jun - 2021


## 06.28 - 07.02
| Title                                                                                                                                                                    |
| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
| [Achieving Statistical Optimality of Federated Learning: Beyond Stationary Points](https://arxiv.org/abs/2106.15216)                                                     |
| [Bottleneck Time Minimization for Distributed Iterative Processes: Speeding Up Gossip-Based Federated Learning on Networked Computers](https://arxiv.org/abs/2106.15048) |
| [Byzantine-robust Federated Learning through Spatial-temporal Analysis of Local Model Updates](https://arxiv.org/abs/2107.01477)                                         |
| [Continual Distributed Learning for Crisis Management](https://arxiv.org/abs/2104.12876)                                                                                 |
| [EasyFL: A Low-code Federated Learning Platform For Dummies](https://arxiv.org/abs/2105.07603)                                                                           |
| [Faithful Edge Federated Learning: Scalability and Privacy](https://arxiv.org/abs/2106.15905)                                                                            |
| [Federated Dynamic Spectrum Access](https://arxiv.org/abs/2106.14976)                                                                                                    |
| [FedFog: Network-Aware Optimization of Federated Learning over Wireless Fog-Cloud Systems](https://arxiv.org/abs/2107.02755)                                             |
| [FedMix: Approximation of Mixup under Mean Augmented Federated Learning](https://arxiv.org/abs/2107.00233)                                                               |
| [Global Knowledge Distillation in Federated Learning](https://arxiv.org/abs/2107.00051)                                                                                  |
| [Gradient-Leakage Resilient Federated Learning](https://arxiv.org/abs/2107.01154)                                                                                        |
| [Multi-task Over-the-Air Federated Learning: A Non-Orthogonal Transmission Approach](https://arxiv.org/abs/2106.14229)                                                   |
| [On Bridging Generic and Personalized Federated Learning](https://arxiv.org/abs/2107.00778)                                                                              |
| [Personalized Federated Learning with Gaussian Processes](https://arxiv.org/abs/2106.15482)                                                                              |
| [PPFL: Privacy-preserving Federated Learning with Trusted Execution Environments](https://arxiv.org/abs/2104.14380)                                                      |
| [Segmented Federated Learning for Adaptive Intrusion Detection System](https://arxiv.org/abs/2107.00881)                                                                 |
| [Towards Scheduling Federated Deep Learning using Meta-Gradients for Inter-Hospital Learning](https://arxiv.org/abs/2107.01707)                                          |
| [Weight Divergence Driven Divide-and-Conquer Approach for Optimal Federated Learning from non-IID Data](https://arxiv.org/abs/2106.14503)                                |

## 06.21 - 06.25
| Title                                                                                                                                       |
| ------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Federated Data-Driven Evolutionary Algorithm for Expensive Multi/Many-objective Optimization](https://arxiv.org/abs/2106.12086)          |
| [A Vertical Federated Learning Framework for Graph Convolutional Network](https://arxiv.org/abs/2106.11593)                                 |
| [Affect-driven Engagement Measurement from Videos](https://arxiv.org/abs/2106.10882)                                                        |
| [Behavior Mimics Distribution: Combining Individual and Group Behaviors for Federated Learning](https://arxiv.org/abs/2106.12300)           |
| [Benchmarking Differential Privacy and Federated Learning for BERT Models](https://arxiv.org/abs/2106.13973)                                |
| [Compositional Federated Learning: Applications in Distributionally Robust Averaging and Meta Learning](https://arxiv.org/abs/2106.11264)   |
| [Enabling Long-Term Cooperation in Cross-Silo Federated Learning: A Repeated Game Perspective](https://arxiv.org/abs/2106.11814)            |
| [FedCM: Federated Learning with Client-level Momentum](https://arxiv.org/abs/2106.10874)                                                    |
| [Federated Graph Classification over Non-IID Graphs](https://arxiv.org/abs/2106.13423)                                                      |
| [Federated Learning with Positive and Unlabeled Data](https://arxiv.org/abs/2106.10904)                                                     |
| [FedFace: Collaborative Learning of Face Recognition Model](https://arxiv.org/abs/2104.03008)                                               |
| [Fed-NILM: A Federated Learning-based Non-Intrusive Load Monitoring Method for Privacy-Protection](https://arxiv.org/abs/2105.11085)        |
| [FedXGBoost: Privacy-Preserving XGBoost for Federated Learning](https://arxiv.org/abs/2106.10662)                                           |
| [FLRA: A Reference Architecture for Federated Learning Systems](https://arxiv.org/abs/2106.11570)                                           |
| [Implicit Gradient Alignment in Distributed and Federated Learning](https://arxiv.org/abs/2106.13897)                                       |
| [Multi-task Over-the-Air Federated Learning: A Non-Orthogonal Transmission Approach](https://arxiv.org/abs/2106.14229)                      |
| [Personalized Federated Learning with Clustered Generalization](https://arxiv.org/abs/2106.13044)                                           |
| [Privacy Threats Analysis to Secure Federated Learning](https://arxiv.org/abs/2106.13076)                                                   |
| [Quantized Federated Learning under Transmission Delay and Outage Constraints](https://arxiv.org/abs/2106.09397)                            |
| [Reward-Based 1-bit Compressed Federated Distillation on Blockchain](https://arxiv.org/abs/2106.14265)                                      |
| [Secure Distributed Training at Scale](https://arxiv.org/abs/2106.11257)                                                                    |
| [STAR-RIS Enabled Heterogeneous Networks: Ubiquitous NOMA Communication and Pervasive Federated Learning](https://arxiv.org/abs/2106.08592) |
| [Subgraph Federated Learning with Missing Neighbor Generation](https://arxiv.org/abs/2106.13430)                                            |
| [Triggerflow: Trigger-based Orchestration of Serverless Workflows](https://arxiv.org/abs/2106.00583)                                        |
| [Understanding Clipping for Federated Learning: Convergence and Client-Level Differential Privacy](https://arxiv.org/abs/2106.13673)        |

## 06.14 - 06.18
| Title                                                                                                                                                                   |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [A Vertical Federated Learning Framework for Horizontally Partitioned Labels](https://arxiv.org/abs/2106.10056)                                                         |
| [A Wasserstein Minimax Framework for Mixed Linear Regression](https://arxiv.org/abs/2106.07537)                                                                         |
| [Accumulative Poisoning Attacks on Real-time Data](https://arxiv.org/abs/2106.09993)                                                                                    |
| [CFedAvg: Achieving Efficient Communication and Fast Convergence in Non-IID Federated Learning](https://arxiv.org/abs/2106.07155)                                       |
| [Coded Federated Learning Framework for AI-Based Mobile Application Services with Privacy-Awareness](https://arxiv.org/abs/2106.09261)                                  |
| [Coded Privacy-Preserving Computation at Edge Networks](https://arxiv.org/abs/2106.08290)                                                                               |
| [Communication-Efficient Agnostic Federated Averaging](https://arxiv.org/abs/2104.02748)                                                                                |
| [CRFL: Certifiably Robust Federated Learning against Backdoor Attacks](https://arxiv.org/abs/2106.08283)                                                                |
| [Decentralized Local Stochastic Extra-Gradient for Variational Inequalities](https://arxiv.org/abs/2106.08315)                                                          |
| [Decentralized Personalized Federated Min-Max Problems](https://arxiv.org/abs/2106.07289)                                                                               |
| [Dynamic Gradient Aggregation for Federated Domain Adaptation](https://arxiv.org/abs/2106.07578)                                                                        |
| [Federated CycleGAN for Privacy-Preserving Image-to-Image Translation](https://arxiv.org/abs/2106.09246)                                                                |
| [Federated Learning for Internet of Things: A Federated Learning Framework for On-device Anomaly Data Detection](https://arxiv.org/abs/2106.07976)                      |
| [Federated Learning for Intrusion Detection System: Concepts, Challenges and Future Directions](https://arxiv.org/abs/2106.09527)                                       |
| [Federated Learning over Energy Harvesting Wireless Networks](https://arxiv.org/abs/2106.08809)                                                                         |
| [Federated Learning with Fair Averaging](https://arxiv.org/abs/2104.14937)                                                                                              |
| [Federated Myopic Community Detection with One-shot Communication](https://arxiv.org/abs/2106.07255)                                                                    |
| [Federated Robustness Propagation: Sharing Adversarial Robustness in Federated Learning](https://arxiv.org/abs/2106.10196)                                              |
| [Federated Semi-supervised Medical Image Classification via Inter-client Relation Matching](https://arxiv.org/abs/2106.08600)                                           |
| [Fine-Grained Data Selection for Improved Energy Efficiency of Federated Edge Learning](https://arxiv.org/abs/2106.12561)                                               |
| [Is Shapley Value fair? Improving Client Selection for Mavericks in Federated Learning](https://arxiv.org/abs/2106.10734)                                               |
| [Locally Differentially Private Federated Learning: Efficient Algorithms with Tight Risk Bounds](https://arxiv.org/abs/2106.09779)                                      |
| [Low-Latency Federated Learning over Wireless Channels with Differential Privacy](https://arxiv.org/abs/2106.13039)                                                     |
| [Multi-VFL: A Vertical Federated Learning System for Multiple Data and Label Owners](https://arxiv.org/abs/2106.05468)                                                  |
| [On Dynamic Resource Allocation for Blockchain Assisted Federated Learning over Wireless Channels](https://arxiv.org/abs/2105.14708)                                    |
| [On Large-Cohort Training for Federated Learning](https://arxiv.org/abs/2106.07820)                                                                                     |
| [Optimal Accounting of Differential Privacy via Characteristic Function](https://arxiv.org/abs/2106.08567)                                                              |
| [Optimality and Stability in Federated Learning: A Game-theoretic Approach](https://arxiv.org/abs/2106.09580)                                                           |
| [Optimized Power Control Design for Over-the-Air Federated Edge Learning](https://arxiv.org/abs/2106.09316)                                                             |
| [Over-the-Air Decentralized Federated Learning](https://arxiv.org/abs/2106.08011)                                                                                       |
| [Privacy Assessment of Federated Learning using Private Personalized Layers](https://arxiv.org/abs/2106.08060)                                                          |
| [Quantized Federated Learning under Transmission Delay and Outage Constraints](https://arxiv.org/abs/2106.09397)                                                        |
| [QuantumFed: A Federated Learning Framework for Collaborative Quantum Training](https://arxiv.org/abs/2106.09109)                                                       |
| [STAR-RIS Enabled Heterogeneous Networks: Ubiquitous NOMA Communication and Pervasive Federated Learning](https://arxiv.org/abs/2106.08592)                             |
| [STEM: A Stochastic Two-Sided Momentum Algorithm Achieving Near-Optimal Sample and Communication Complexities for Federated Learning](https://arxiv.org/abs/2106.10435) |
| [Towards Heterogeneous Clients with Elastic Federated Learning](https://arxiv.org/abs/2106.09433)                                                                       |
| [Zero-Shot Federated Learning with New Classes for Audio Classification](https://arxiv.org/abs/2106.10019)                                                              |

## 06.07 - 06.11
| Title                                                                                                                                                                         |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| [Adaptive Dynamic Pruning for Non-IID Federated Learning](https://arxiv.org/abs/2106.06921)                                                                                   |
| [Cross-Node Federated Graph Neural Network for Spatio-Temporal Data Modeling](https://arxiv.org/abs/2106.05223)                                                               |
| [Data-Free Knowledge Distillation for Heterogeneous Federated Learning](https://arxiv.org/abs/2105.10056)                                                                     |
| [D-Cliques: Compensating NonIIDness in Decentralized Federated Learning with Topology](https://arxiv.org/abs/2104.07365)                                                      |
| [Differentially Private Federated Learning via Inexact ADMM](https://arxiv.org/abs/2106.06127)                                                                                |
| [DP-NormFedAvg: Normalizing Client Updates for Privacy-Preserving Federated Learning](https://arxiv.org/abs/2106.07094)                                                       |
| [Efficient and Less Centralized Federated Learning](https://arxiv.org/abs/2106.06627)                                                                                         |
| [Exploiting Record Similarity for Practical Vertical Federated Learning](https://arxiv.org/abs/2106.06312)                                                                    |
| [Fast Federated Learning in the Presence of Arbitrary Device Unavailability](https://arxiv.org/abs/2106.04159)                                                                |
| [FedDICE: A ransomware spread detection in a distributed integrated clinical environment using federated learning and SDN based mitigation](https://arxiv.org/abs/2106.05434) |
| [Federated Hyperparameter Tuning: Challenges, Baselines, and Connections to Weight-Sharing](https://arxiv.org/abs/2106.04502)                                                 |
| [Federated Learning of User Verification Models Without Sharing Embeddings](https://arxiv.org/abs/2104.08776)                                                                 |
| [Federated Learning on Non-IID Data: A Survey](https://arxiv.org/abs/2106.06843)                                                                                              |
| [Federated Learning Over Wireless Channels: Dynamic Resource Allocation and Task Scheduling](https://arxiv.org/abs/2106.06934)                                                |
| [Federated Learning with Buffered Asynchronous Aggregation](https://arxiv.org/abs/2106.06639)                                                                                 |
| [Federated Learning with Spiking Neural Networks](https://arxiv.org/abs/2106.06579)                                                                                           |
| [FedNILM: Applying Federated Learning to NILM Applications at the Edge](https://arxiv.org/abs/2106.07751)                                                                     |
| [FedNLP: An interpretable NLP System to Decode Federal Reserve Communications](https://arxiv.org/abs/2106.06247)                                                              |
| [FedScale: Benchmarking Model and System Performance of Federated Learning](https://arxiv.org/abs/2105.11367)                                                                 |
| [FGLP: A Federated Fine-Grained Location Prediction System for Mobile Users](https://arxiv.org/abs/2106.08946)                                                                |
| [Gradient Disaggregation: Breaking Privacy in Federated Learning by Reconstructing the User Participant Matrix](https://arxiv.org/abs/2106.06089)                             |
| [Incentive Mechanism for Privacy-Preserving Federated Learning](https://arxiv.org/abs/2106.04384)                                                                             |
| [Joint Client Scheduling and Resource Allocation under Channel Uncertainty in Federated Learning](https://arxiv.org/abs/2106.06796)                                           |
| [Memory-based Optimization Methods for Model-Agnostic Meta-Learning](https://arxiv.org/abs/2106.04911)                                                                        |
| [No Fear of Heterogeneity: Classifier Calibration for Federated Learning with Non-IID Data](https://arxiv.org/abs/2106.05001)                                                 |
| [Rethinking Architecture Design for Tackling Data Heterogeneity in Federated Learning](https://arxiv.org/abs/2106.06047)                                                      |
| [Securing Secure Aggregation: Mitigating Multi-Round Privacy Leakage in Federated Learning](https://arxiv.org/abs/2106.03328)                                                 |
| [Understanding the Interplay between Privacy and Robustness in Federated Learning](https://arxiv.org/abs/2106.07033)                                                          |
| [Unsupervised Clustered Federated Learning in Complex Multi-source Acoustic Environments](https://arxiv.org/abs/2106.03671)                                                   |
| [User Label Leakage from Gradients in Federated Learning](https://arxiv.org/abs/2105.09369)                                                                                   |
| [Vertical Federated Learning without Revealing Intersection Membership](https://arxiv.org/abs/2106.05508)                                                                     |
